client/src/components/logout/logout.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './logout.css';
+
+const Logout = () => {
```

Comment:
I think you don't need this component. After logout user should be redirected to login page.